source/codegen/metadata/nisync/functions.py

```diff
@@ -16,4 +16,283 @@
+                "type": "ViConstString"
+            },
+            {
+                "direction": "in",
+                "name": "syncClock",
+                "type": "ViConstString"
+            },
+            {
+                "direction": "in",
+                "name": "invert",
+                "type": "ViInt32"
+            },
+            {
+                "direction": "in",
+                "name": "updateEdge",
+                "type": "ViInt32"
+            },
+            {
+                "direction": "in",
+                "name": "delay",
+                "type": "ViReal64"
+            }
+        ],
+        "returns": "ViStatus"
+    },
+    "DisconnectSWTrigFromTerminal": {
+        "parameters": [
+            {
+                "direction": "in",
+                "name": "vi",
+                "type": "ViSession"
+            },
+            {
+                "direction": "in",
+                "name": "srcTerminal",
+                "type": "ViConstString"
+            },
+            {
+                "direction": "in",
+                "name": "destTerminal",
+                "type": "ViConstString"
+            }
+        ],
+        "returns": "ViStatus"
+    },
+    "ConnectTrigTerminals": {
+        "parameters": [
+            {
+                "direction": "in",
+                "name": "vi",
+                "type": "ViSession"
+            },
+            {
+                "direction": "in",
+                "name": "srcTerminal",
+                "type": "ViConstString"
+            },
+            {
+                "direction": "in",
+                "name": "destTerminal",
+                "type": "ViConstString"
+            },
+            {
+                "direction": "in",
+                "name": "syncClock",
+                "type": "ViConstString"
+            },
+            {
+                "direction": "in",
+                "name": "invert",
+                "type": "ViInt32"
+            },
+            {
+                "direction": "in",
+                "name": "updateEdge",
+                "type": "ViInt32"
+            }
+        ],
+        "returns": "ViStatus"
+    },
+    "DisconnectTrigTerminals": {
+        "parameters": [
+            {
+                "direction": "in",
+                "name": "vi",
+                "type": "ViSession"
+            },
+            {
+                "direction": "in",
+                "name": "srcTerminal",
+                "type": "ViConstString"
+            },
+            {
+                "direction": "in",
+                "name": "destTerminal",
+                "type": "ViConstString"
+            }
+        ],
+        "returns": "ViStatus"
+    },
+    "GetAttributeViInt32": {
+        "parameters": [
+            {
+                "direction": "in",
+                "name": "vi",
+                "type": "ViSession"
+            },
+            {
+                "direction": "in",
+                "name": "terminalName",
```

Comment:
Use activeItem instead of terminalName since it is not always a terminal. Same for other occurrences.

Reply:
done